domain/src/main/java/com/teamagam/gimelgimel/domain/messages/entity/contents/VectorLayer.java

```diff
@@ -0,0 +1,44 @@
+package com.teamagam.gimelgimel.domain.messages.entity.contents;
+
+import java.net.URI;
+import java.net.URL;
+
+public class VectorLayer {
+
+    private final String mId;
+    private String mName;
```

Comment:
I used the final qualifier just for just the ID because as I see it, the ID is the only thing that cannot change in a Vector. An update can be sent from the server to change the layer's name or actual layer (remoteUrl).